Makefile

```diff
@@ -119,6 +120,17 @@ endif
 	rm -f test/copyimg/copyimg
 	rm -f test/checkseccomp/checkseccomp
 
+
+local-cross:
+	@for target in windows/amd64 darwin/amd64 ; do \
```

Comment:
Should there also be a `linux/amd64` target so folks building `local-cross` on Windows and OS X cross-compile the Windows binaries?  For an example of handling this issue in a sibling project, see [here][1] [here][2], [here][3], and [here][4].

[1]: https://github.com/kubernetes-incubator/bootkube/blob/b546d45fbc4e430e616ad50961e400f6f0db2b2e/Makefile#L6
[2]: https://github.com/kubernetes-incubator/bootkube/blob/b546d45fbc4e430e616ad50961e400f6f0db2b2e/Makefile#L11-L12
[3]: https://github.com/kubernetes-incubator/bootkube/blob/b546d45fbc4e430e616ad50961e400f6f0db2b2e/Makefile#L16-L24
[4]: https://github.com/kubernetes-incubator/bootkube/blob/b546d45fbc4e430e616ad50961e400f6f0db2b2e/Makefile#L45-L50